lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -6811,4 +6846,4 @@ void BoundInterpreterFunction::fwdBBoxTransformInst(
 void BoundInterpreterFunction::fwdExternalFunctionCallInst(
     glow::ExternalFunctionCallInst const *) {
   LOG(FATAL) << "ExternalFunctionCallInst is not supported yet";
-}
+}
```

Comment:
@vbaddi Sorry one last thing -- can you add back the newline here? Our internal linters don't like this and it's causing me some issues.

Reply:
Done